### ISSUE TITLE
main thread panics if you pass an absolute path to import()

### ISSUE THREAD

**Issue by envis10n** [denoland/deno#4641]
I had made an error with the way I was setting up the path for dynamic imports, and it was crashing showing a thread panic on the rust side. I checked the path, and it was using the actual absolute path on windows (C:\...). I fixed this to use the relative path, and it loads fine. The problem, in my opinion, is that it panics when the path provided is not using the file:/// prefix, or a relative path.

**Comment by kitsonk** [comment]
Can you provide the actual panic?

**Comment by envis10n** [comment]
[Code]
```ts
const imp = await import(`${Deno.cwd()}\\commands\\echo.ts`);

console.log(imp);
```

[Panic]
```
thread 'main' panicked at 'internal error: entered unreachable code', cli\state.rs:361:12
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by kitsonk** [comment]
@envis10n thanks!

It is a bug, this:

https://github.com/denoland/deno/blob/6720a0dc02dc5a93472520e6d23ca033d7ec9d88/cli/state.rs#L372

Should return an `OpError` instead of using the unreachable macro.  Should be an easy fix for someone if they wanted to contribute to Deno.

**Comment by bartlomieju** [comment]
Actually it might be related to https://github.com/denoland/deno/issues/3355. The `unreachable!()` statement is there for a reason - it shouldn't ever be hit as "previous" layers should resolve or reject the invalid specifier.

**Comment by bartlomieju** [comment]
Actually it's definitely #3355 I had encountered this problem a while ago - quick solution was to prefix the import with `file://`, but we need to fix the underlying problem.

**Comment by bartOssh** [comment]
@bartlomieju shall I remove [ #4646](https://github.com/denoland/deno/pull/4646) PR and work it out other way ?

**Comment by nayeemrmn** [comment]
Looks fixed in 1.7.4:
```
error: Uncaught (in promise) TypeError: Unsupported scheme "c" for module "c:\Users\Nayeem\projects\deno\temp2.ts". Supported schemes: [
    "data",
    "file",
    "http",
    "https",
]
```
